README.md

```diff
@@ -99,7 +99,7 @@ $ rails console
 >
 > v = Volunteer.new
 > v.email = "jane.doe@gmail.com"
-> v.password "changeme"
+> v.password = "changeme"
```

Comment:
Can you also add that v.password_confirmation = "changeme" ? I believe that's a required step that isn't in here.